bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/protocol/RemoteControllerWebSocket.java

```diff
@@ -55,37 +64,33 @@ public class RemoteControllerWebSocket extends RemoteController implements Liste
     private final WebSocketArt webSocketArt;
     private final WebSocketV2 webSocketV2;
 
+    // refresh limit for current app update (in seconds)
+    private static final long UPDATE_CURRENT_APP_REFRESH = 10;
```

Comment:
To perhaps simplify even further, you could:
```suggestion
    private static final Duration UPDATE_CURRENT_APP_REFRESH = Duration.ofSeconds(10);
```

that would even make the comment above redundant. Instead of `plusSeconds`, you could then simply use `plus`.

Reply:
I think it's more readable as is.

Reply:
Two other suggestions then if you prefer to keep a `long` with number of seconds:
1. Promote the comment above to Javadoc comment so IDE's could show it on mouse-over and reveal that it's intended to contain number of seconds.
2. Rename to `UPDATE_CURRENT_APP_REFRESH_SECONDS`.

Or a combination. :)

Reply:
Renamed to UPDATE_CURRENT_APP_REFRESH_SECONDS